onnxruntime/test/contrib_ops/matmul_4bits_test.cc

```diff
@@ -63,7 +63,7 @@ void QuantizeDequantize(std::vector<float>& raw_vals,
       tp.get());
 }
 
-void RunTest(int64_t M, int64_t N, int64_t K, int64_t block_size, MLAS_SQNBIT_COMPUTE_TYPE comp_type,
+void RunTest(int64_t M, int64_t N, int64_t K, int64_t block_size, int64_t comp_type,
```

Comment:
nit: please also use the enum here instead of int64_t

Reply:
This test is created for all MatMulNBits implementations. But the enum only exists with neural_speed on. It's better to change the parameter name to accuracy_level here and keep using int64_t. Or do I need to define an operator-level enum, shared by neural_speed and MLAS?

Reply:
the concern I have with just using integer literals is that it may not be obvious what they mean. ideally there would be a single enum that represents the compute type.

but it's not a blocker for this PR, we can it clean up later.

Reply:
resolved as it's not a blocker.